iphone/Classes/TiUIiOSProxy.m

```diff
@@ -130,9 +130,23 @@ - (void)setStatusBarBackgroundColor:(id)value
   ENSURE_UI_THREAD(setStatusBarBackgroundColor, value);
   ENSURE_SINGLE_ARG(value, NSString);
 
-  UIView *statusBar = [[[UIApplication sharedApplication] valueForKey:@"statusBarWindow"] valueForKey:@"statusBar"];
-  if ([statusBar respondsToSelector:@selector(setBackgroundColor:)]) {
-    statusBar.backgroundColor = [[TiUtils colorValue:value] _color];
+  if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
+#if IS_SDK_IOS_13
+    CGRect frame = [[[[[UIApplication sharedApplication] keyWindow] windowScene] statusBarManager] statusBarFrame];
```

Comment:
The docs say `keyWindow` is deprecated since iOS 13 for applications that support multiple scenes. I'm not quite sore if we support that?

Please use property accessors where possible to avoid all the brackets.

```suggestion
    CGRect frame = UIApplication.sharedApplication.keyWindow.windowScene.statusBarManager.statusBarFrame;
```

Reply:
Multiple scenes are not yet supported in SDK.